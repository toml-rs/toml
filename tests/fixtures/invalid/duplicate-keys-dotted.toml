[http]
ssl-version = 'tlsv1.1'
ssl-version.min = 'tlsv1.2'
ssl-version.max = 'tlsv1.3'
