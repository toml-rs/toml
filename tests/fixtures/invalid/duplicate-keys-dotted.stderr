TOML parse error at line 3, column 1
  |
3 | ssl-version.min = 'tlsv1.2'
  | ^
Dotted key `ssl-version` attempted to extend non-table type (string)

